src/Crypto/WebAuthn/Model/Types.hs

```diff
@@ -659,26 +660,55 @@ newtype PublicKeyBytes = PublicKeyBytes {unPublicKeyBytes :: BS.ByteString}
 -- | [(spec)](https://www.w3.org/TR/webauthn-2/#iface-authentication-extensions-client-inputs)
 -- This is a dictionary containing the [client extension input](https://www.w3.org/TR/webauthn-2/#client-extension-input)
 -- values for zero or more [WebAuthn Extensions](https://www.w3.org/TR/webauthn-2/#webauthn-extensions).
--- TODO: Extensions are not implemented by this library, see "Crypto.WebAuthn.Model.Types#extensions".
-data AuthenticationExtensionsClientInputs = AuthenticationExtensionsClientInputs
-  {
+-- TODO: Most extensions are not implemented by this library, see "Crypto.WebAuthn.Model.Types#extensions".
+newtype AuthenticationExtensionsClientInputs = AuthenticationExtensionsClientInputs
+  { -- | [(spec)](https://www.w3.org/TR/webauthn-2/#sctn-authenticator-credential-properties-extension)
+    -- When true, indicates that that extension is requested by the [Relying Party](https://www.w3.org/TR/webauthn-2/#relying-party).
+    aeciCredProps :: Maybe Bool
   }
   deriving (Eq, Show)
```

Comment:
```suggestion
  deriving (Eq, Show, ToJSON)
```

This should work, same for `CredentialPropertiesOutput` and `AuthenticationExtensionsClientOutputs`

Reply:
Done!